### ISSUE TITLE
Glossary doesn't match text

### ISSUE THREAD

**Issue by ecayer** [GSA/digitalgov-pra#130]
In /do-i-need-clearance/form-updates-and-conversions/

The text on the page is " OIRA would likely consider conversion to an electronic format a non-substantive change to an already approved collection, and it wouldn’t need additional approval."  

However, if you click on "non-substantive" in that phrase, it links to a glossary item that says, "A change made to an information collection that requires OIRA review and approval but does not require public comment."

In short, text says "doesn't need approval" and glossary says, "does need approval." These should be made consistent.
